test/e2e/route_advertisements.go

```diff
@@ -222,3 +226,233 @@ var _ = ginkgo.Describe("BGP: Pod to external server when default podNetwork is
 		})
 	})
 })
+
+var _ = ginkgo.Describe("BGP: Pod to external server when UDN Layer3 Network is advertised", func() {
+	const (
+		serverContainerName    = "bgpserver"
+		routerContainerName    = "frr"
+		echoClientPodName      = "echo-client-pod"
+		echoServerPodPortMin   = 9800
+		echoServerPodPortMax   = 9899
+		primaryNetworkName     = "kind"
+		bgpExternalNetworkName = "bgpnet"
+		testUdnName            = "bgp-udn-layer3-network"
+		testRAName             = "udn-layer3-ra"
+	)
+	var serverContainerIPs []string
+	var frrContainerIPv4, frrContainerIPv6 string
+	var nodes *corev1.NodeList
+	var cs clientset.Interface
+	f := wrappedTestFramework("pod2external-route-advertisements")
+	// disable automatic namespace creation, we need to add the required UDN label
+	f.SkipNamespaceCreation = true
+
+	if !IsGatewayModeLocal() {
+		const upstreamIssue = "https://github.com/ovn-kubernetes/ovn-kubernetes/issues/5134"
+		e2eskipper.Skipf(
```

Comment:
can you skip here? thought it needed to be within a bottom ginkgo node>

Reply:
lol you are right! how did this work for me downstream?

Reply:
not sure - for me before, it used to be a manged ginkgo panic iirc